jvm-packages/dev/Dockerfile

```diff
@@ -21,10 +21,10 @@ FROM centos:7
 # Install all basic requirements
 RUN \
     yum -y update && \
-    yum install -y bzip2 make tar unzip wget xz git centos-release-scl yum-utils java-1.8.0-openjdk-devel && \
+    yum install -y which bzip2 make tar unzip wget xz git centos-release-scl yum-utils java-1.8.0-openjdk-devel && \
     yum-config-manager --enable centos-sclo-rh-testing && \
     yum -y update && \
-    yum install -y devtoolset-7-gcc devtoolset-7-binutils devtoolset-7-gcc-c++ && \
+    yum install -y devtoolset-7-gcc devtoolset-7-binutils devtoolset-7-gcc-c++ gcc-gfortran python3-devel openblas-devel lapack-devel pkgconfig  && \
     # Python
```

Comment:
Not sure if we need to change the docker file here.

Reply:
I follow the README.md in jvm-package.
./xgboost/jvm-packages/dev/build-linux.sh will fail for 
1. there is no `which` command 
2. with openblas-devel `scipy` can not be install